resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
https://www.freeformatter.com/html-formatter.html

please use this to format the HTML file. This look so bad. Also apply for other HTML files.